sql/moz-fx-data-shared-prod/fenix_derived/feature_usage_events_v1/schema.yaml

```
@@ -2,7 +2,19 @@ fields:
 - name: submission_date
   type: DATE
   mode: NULLABLE
-- name: events_ping_distinct_client_count
+- name: ping_date
+  type: DATE
+  mode: NULLABLE
+- name: channel
+  type: STRING
+  mode: NULLABLE
+- name: country
+  type: STRING
+  mode: NULLABLE
+- name: adjust_network
+  type: STRING
+  mode: NULLABLE
+- name: is_default_browser
   type: INTEGER
```

```suggestion
  type: BOOLEAN
```

Shouldn't `is_default_browser` be a BOOLEAN field?